### PR TITLE
add Turkish translation

### PR DESCRIPTION
**Pull request by mrtrkmn**
Turkish translation is added.

### CODE REVIEW

File: i18n/tr.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  translation: "Çeviriler"
+
+- id: home
+  translation: "Ana Sayfa"
+
+- id: code_copy
+  translation: "Kopyala"
+
+- id: code_copied
+  translation: "Kopyalandı !"
```
Comment (adityatelange) [inline i18n/tr.yaml:25]: Also the space before `!` should't be here I think ?

File: i18n/tr.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    other: "{{ .Count }} dk"
+
+- id: toc
+  translation: "İçindekiler"
+
+- id: translations
+  translation: "Çeviriler"
+
+- id: home
+  translation: "Ana Sayfa"
```
[the remark this answers is not in the document]
Comment (mrtrkmn) [reply i18n/tr.yaml:19]: Hmm the translation is not very well for web context. " Ev " has different meaning and not used in web context.

File: i18n/tr.yaml (hunk 1 above)
Comment (mrtrkmn) [reply i18n/tr.yaml]: Updated.

File: i18n/tr.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrtrkmn) [reply i18n/tr.yaml]: updated.

**Review by adityatelange — APPROVED** [review]
Thanks !

**Pull request merged**